docs/resources/mws_workspaces.md

```diff
@@ -50,7 +50,7 @@ resource "databricks_mws_networks" "this" {
   account_id   = var.databricks_account_id
   network_name = "${var.prefix}-network"
   vpc_id       = var.vpc_id
-  subnet_ids = [var.subnet_public, var.subnet_private]
+  subnet_ids = [var.subnet_private]
```

Comment:
workspace requires at least two subnets, did you verify this change to work?

Reply:
actually, in my tests I was using multiple private subnets since the best practices are to always use multiple AZs. 

do you want me to change the whole VPC definition?

Reply:
changing expected variable type to list (variable definition not present, so just changing the name and the syntax).